examples/77_blackwell_fmha/collective/fmha_fusion.hpp

```diff
@@ -225,8 +225,8 @@ struct CausalMask : NoMask {
     if constexpr (IsQBegin) {
       return std::min(trip_count, int(ceil_div(size<0>(tile_shape), size<1>(tile_shape))));
     } else {
-      const int corner_count = int((get<1>(problem_size) % get<1>(tile_shape) || get<0>(problem_size) % get<0>(tile_shape))) ; 
-      return std::min(trip_count, int(ceil_div(get<0>(tile_shape), get<1>(tile_shape))) + corner_count);
+      int offset_tile_q = (get<1>(problem_size) % get<1>(tile_shape)) - (get<0>(problem_size) % get<1>(tile_shape));
```

Comment:
why is this dividing 0 problem size by 1 tile shape? Are there any implicitinvariants, like `get<1>(tile_shape)` is a multiple of `get<0>(tile_shape)`? If so, they should be explicitly mentioned in the comments otherwise this is very confusing.